Makefile

```diff
@@ -1,3 +1,5 @@
+DHOST = $(shell echo $$(docker-machine ip))
```

Comment:
I've added code to the makefile that I've tested as working on both Mac and Linux (Ubuntu), based on info on this StackOverflow article: http://stackoverflow.com/questions/3466166/how-to-check-if-running-in-cygwin-mac-or-linux

If I need to consider other types of hosts, pet me know and I'll update the PR.